.parcelrc

```diff
@@ -0,0 +1,7 @@
+{
+	"extends": "@parcel/config-default",
+  "resolvers": ["@parcel/resolver-glob", "..."],
+   "transformers": {
+    "*.hbs": ["parcel-transformer-hbs"]
+  }
+}
```

Comment:
[EOF](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

https://eslint.org/docs/rules/eol-last